src/app/App.tsx

```diff
@@ -51,7 +51,7 @@ BigNumber.config({
   },
 })
 
-interface Props {
+interface Props extends Record<string, unknown> {
```

Comment:
this is to satisfy Sentry's updated `wrap` types (we use `Sentry.wrap(App)` below)

Reply:
surprisingly I don't see TS complaining if I remove that

Reply:
oh weird, you can see the unit test / iOS and android test jobs failing on the CI in [this commit](https://github.com/valora-inc/wallet/pull/5319/commits/ce81c56aad0bc373799f210fe0e83288b2f6973a) which doesn't have this change. perhaps you need to update dependencies and restart vs code? i got into a weird state with TS in vs code a few times doing all these upgrades

Reply:
ah, I had to restart vs code. now it yells!